### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by momack2**

### CODE REVIEW

**Comment by walkerford** [comment]
I added in my updates to this file.  My auto-formatter made the new commit seem like more changed than actually did change.

File: README.md [hunk 1]
```diff
@@ -1,73 +1,106 @@
...
-* [Filecoin](docs/filecoin/README.md)
+### Tutorials
 
-### Dev Tools
-* [Protocol Labs Toolkits & SDKs](docs/protocol-labs-toolkits-sdks/README.md)
-* [Built on IPFS and Filecoin](docs/built-on-ipfs-filecoin/README.md)
-* [Additional Learning](docs/additional-learning-resources/README.md)
+- [FVM](content/en/tutorials)
+- [IPFS](content/en/ipfs-intro)
+- [IPFS](content/en/ipld)
```
Comment (momack2) [inline README.md:49]:
```suggestion
- [IPLD](content/en/ipld)
```

File: README.md [hunk 2]
```diff
@@ -1,73 +1,106 @@
...
-* Curriculum website structure, formatting, and maintenance: @chase
-* PL Network: 
-* web3 Basics: 
-* IPFS: 
-* IPLD:
-* libp2p: 
-* Filecoin: 
-* DevTools: 
-* Launchpad Getting Started: @momack2
+- Curriculum website structure, formatting, and maintenance: @chase
```
Comment (momack2) [inline README.md:77]:
```suggestion
### Content Maintainers

When editing content, please ask for a PR review from a content maintainer:
- Curriculum website structure, formatting, and maintenance: @chase
```

**Comment by momack2** [comment]
re-reviewed - need someone else to approve this pls

File: README.md [hunk 3]
```diff
@@ -1,73 +1,109 @@
...
+- Launchpad Getting Started: @momack2
+- Web3 Basics:
+- PL Network: @rschanger
+- IPFS:
+- IPLD:
+- libp2p:
+- Filecoin:
+- DevTools:
```
Comment (momack2) [inline README.md:88]:
```suggestion
- Web3 Basics: @hammertoe
- PL Network: @rschanger 
- IPFS: @2color
- IPLD: @2color
- libp2p: @2color
- Filecoin: @hammertoe
- DevTools: @hammertoe
```

Comment (momack2) [reply README.md:88]: @kt-wawro to review this part please!

Comment (momack2) [reply README.md:88]: (and make sure to give edit/merge access to the github repo)

**Pull request merged**